hbase-client/src/main/java/org/apache/hadoop/hbase/client/MetricsConnection.java

```diff
@@ -443,6 +447,15 @@ protected Ratio getRatio() {
     this.nsLookups = registry.counter(name(this.getClass(), NS_LOOKUPS, scope));
     this.nsLookupsFailed = registry.counter(name(this.getClass(), NS_LOOKUPS_FAILED, scope));
 
+    this.userRegionLockTimeoutCount =
+      registry.counter(name(this.getClass(), "userRegionLockTimeoutCount", scope));
+    this.userRegionLockWaitingTimer =
+      registry.timer(name(this.getClass(), "userRegionLockWaitingTimer", scope));
+    this.userRegionLockHeldTimer =
+      registry.timer(name(this.getClass(), "userRegionLockHeldTimer", scope));
+    this.userRegionLockQueueHist =
+      registry.histogram(name(MetricsConnection.class, "userRegionLockQueueHist", scope));
```

Comment:
consider not including `Hist` - since it will be appended with other histogram related stuff - and call it length instead
```suggestion
      registry.histogram(name(MetricsConnection.class, "userRegionLockQueueLength", scope));
```